### PR TITLE
All Assignment Added

### PR DESCRIPTION
**Pull request by pradip0852**

### CODE REVIEW

**Review by Manishnerkar — APPROVED** [review]

**Review by santoshinawkhare — APPROVED** [review]

**Review by santoshinawkhare — APPROVED** [review]

**Review by Desai8595 — APPROVED** [review]

File: Assignments/HTML_CSS/Assign2/pradip/pradipA2.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <hr>
+        <h1>Initial public offering</h1>  
+        <p>Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004.</p>
+        <p>At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins>
+            years, until the year 2024.</p>
+        <marquee behavior="alternate" direction= "left" scrolldelay=5 >
+        <h1 style="color: forestgreen;">Thank You</h1>
+        </marquee>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign2/pradip/pradipA2.html:26]: add new line

File: Assignments/HTML_CSS/Assign3/Pradip/pradipA3_1.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+     </ul>
+      <li><a1> SYSTEM-2</li>
+     <ul><a1>
+      <li><a1>triveini</li>
+      <li><a1>kalyani</li>
+     </ul>
+     </ol>
+     </ol>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/Pradip/pradipA3_1.html:54]: add new line 
need indentation

File: Assignments/HTML_CSS/Assign3/Pradip/pradipA3_2contact.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+ <html>
+  <head>
+      <title>Question2</title>
+  </head>
+
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/Pradip/pradipA3_2contact.html:5]: remove extra lines

File: Assignments/HTML_CSS/Assign3/Pradip/pradipA3_2contact.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+          Contact Us.  Contact Us.  Contact Us.  Contact Us. 
+          Contact Us.  Contact Us.  Contact Us.  Contact Us.
+          Contact Us.  Contact Us.  Contact Us.  Contact Us. 
+          Contact Us.  Contact Us.  Contact Us.  Contact Us. 
+          Contact Us.  Contact Us.  Contact Us.  Contact Us. 
+          Contact Us.  Contact Us.  Contact Us.  Contact Us.  
+      </p>
+      </h3>
+</body>
+</html>
```
Comment (Pawan1005) [inline Assignments/HTML_CSS/Assign3/Pradip/pradipA3_2contact.html:33]: need indentation 
remove extra lines

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**